improver/wxcode/utilities.py

```diff
@@ -117,16 +116,14 @@ def update_daynight(cubewx):
     """ Update weather cube depending on whether it is day or night
```

Comment:
Just out of reach of code suggestions. The interface above needs types: `(cubewx: Cube) -> Cube`

Reply:
I can't add a type annotation here as it causes `test_import_cli` in `improver_tests/cli/test_init.py` to fail.